### ISSUE TITLE
why use add operation in TimeBlock layer

### ISSUE THREAD

**Issue by shirenyu** [FelixOpolka/STGCN-PyTorch#6]
In the tensorflow implementation, the temporal_conv_layer result is the product of conv and sigmoid.
 “ return (x_conv[:, :, :, 0:c_out] + x_input) * tf.nn.sigmoid(x_conv[:, :, :, -c_out:]) ”

could you explain why the add operation is used here?

temp = self.conv1(X) + torch.sigmoid(self.conv2(X))
out = F.relu(temp + self.conv3(X))

**Comment by NotEnded99** [comment]
Same question as you

**Comment by lp2900** [comment]
Yeah, no GLU .

**Comment by ManlioWu** [comment]
I have the same question: Why addition rather than multiplication?
